keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2663,12 +2664,24 @@ Start of v1.32 development cycle (v1.32.0-alpha.1-178-gd9c46d8ecb1):
 
 - `k8s.io/dynamic-resource-allocation/cel`: 88.8%
 - `k8s.io/dynamic-resource-allocation/structured`: 82.7%
+- `k8s.io/kubernetes/pkg/apis/resource/validation`: 77.1%
 - `k8s.io/kubernetes/pkg/controller/resourceclaim`: 70.0%
 - `k8s.io/kubernetes/pkg/kubelet/cm/dra`: 78.6%
 - `k8s.io/kubernetes/pkg/kubelet/cm/dra/plugin`: 77.7%
 - `k8s.io/kubernetes/pkg/kubelet/cm/dra/state`: 46.2%
 - `k8s.io/kubernetes/pkg/scheduler/framework/plugins/dynamicresources`: 72.9%
 
+End of v1.32 development cycle (https://github.com/kubernetes/kubernetes/pull/127511 = v1.32.0-alpha.3-283-g886928b7080):
```

Comment:
WIP because this PR hasn't been merged yet.

Reply:
Updated after that PR got merged, ready for approval.

/assign @johnbelamaric 
/cc @alculquicondor